metrics/plugins/prometheus/README.md

```diff
@@ -26,3 +26,18 @@ local httpd = require('http.server').new(...)
 ...
 httpd:route( { path = '/metrics' }, prometheus.collect_http)
```

Comment:
Here is an example of http.server v1, in your patch you use http v2. I think both parts should use one version.

Reply:
```lua
local httpd = require('http.server').new(...)
local router = require('http.router').new(...)
httpd:set_router(router)
...
router:route( { path = '/metrics' }, prometheus.collect_http)
```